Model/src/main/java/org/gusdb/wdk/model/user/dataset/event/model/EventRow.java

```diff
@@ -0,0 +1,84 @@
+package org.gusdb.wdk.model.user.dataset.event.model;
+
+import java.time.LocalDateTime;
+
+import org.gusdb.wdk.model.user.dataset.UserDatasetType;
+
+/**
+ * Class modeling a single row in the "userdatasetevent" table.
```

Comment:
> I like this class in general. It could do a bit more work if it had a private constructor and factory methods: newEvent with new date and processing status, and fromUserDatasetEvent that takes UserDatasetEvent as an argument

I could see having a separate builder or function that does what you describe, but I'm not a fan of mixing concerns like that.  The EventRow class is simply modeling a DB row, and IMO has no reason to know about anything outside of that scope.  However, I might consider instead having a constructor method that takes a `ResultSet` and constructs a new instance.